versioned_docs/version-3.x/developer/community/zero-downtime-migrations.mdx

```diff
@@ -15,6 +15,23 @@ To achieve the goal we need to ensure that:
 - Old code is compatible with new database schema or new database schema is compatible with old code.
 - Migrations don't lock tables, rows etc. for no longer than one second.
 
+## Task Autodiscovery in Zero Downtime Migrations
+
+In the context of zero downtime migrations, especially for data migrations, tasks might be scattered across different modules or might be version-specific. To ensure that Celery picks up all necessary tasks for execution without missing any, we use the concept of "Additional Autodiscovery with Specified Related Names."
+
+### Additional Autodiscovery with Specified Related Names
+
+This method of autodiscovery allows for targeted discovery of tasks from specific packages or modules, ensuring that migration tasks specific to certain versions or modules are correctly discovered and executed. For instance:
```

Comment:
I think we can even mention exact file that needs to be modified here.

Reply:
done